content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,158 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating `cypress-file-upload` to [`.selectFile()`](/api/commands/selectfile)
+
+Selecting files with input elements or dropping them over the page is available
+in Cypress 9.3. Read the [`.selectFile()` API docs](/api/commands/selectfile)
+for more information on how this works and how to use it. This guide details how
+to change your test code to migrate from
```

Comment:
Can we say "migrate from the cypress-file-upload plugin" instead of "migrate from cypress-file-upload" here?

Reply:
Done.